src/studio/src/designer/frontend/packages/schema-editor/src/components/InputField.tsx

```diff
@@ -66,7 +66,9 @@ export function InputField(props: IInputFieldProps) {
   };
 
   const onBlurKey = (e: any) => {
-    props.onChangeKey(props.fullPath, props.value, e.target.value);
+    if (value !== props.value) {
```

Comment:
I think we could make it a little bit clearer what 'value' we are talking about in this file.
Is 'Key' and 'Value' the same and if so could we make them a little bit more cohesive in terms of naming?

Reply:
Cleaned up InputFIeld